modules/learn/pages/security/on-the-wire-security.adoc

```diff
@@ -63,9 +63,9 @@ The relationships between _global_ and _per service_ settings for TLS and cipher
 
 The minimum TLS version can be established either globally or per service: the server will not accept client connections with protocols below the level of the established, minimum version.
 
-The minimum version can be established globally and for all services as `tlsv1`, `tlsv1.1`, or `tlsv1.2`.
-A value of `tlsv1.3` can be established for the following services: _Data_, _Search_, _Index_, _Query_, _Eventing_, _Analytics_, and _Backup_; it _cannot_ be established either globally or for the Cluster Manager.
-See xref:learn:security/on-the-wire-security.adoc#cipher-suite-configuration-limitations[Cipher-Suite Configuration Limitations], below, for further information.
+The minimum version can be established globally and for all services as `tlsv1`, `tlsv1.1`, `tlsv1.2`.
```

Comment:
Should there be a trailing period after this but before the .. "or `tlsv1.3`." part?

Reply:
Yes, thank you.